CONTINUITY.md

```
@@ -1,78 +1,76 @@
-Tests verdes (`test_cfdi_moneda_extranjera` 10/10, `test_ffm_nace_error_fiscal_event` 5/5,
-regresión `test_cancelacion_integridad` 37 / `test_e4_puente_si_pac` 17) + `ruff` + `mkdocs
---strict` limpios + validación GUI/sandbox (USD `FFMX-2026-00299`: `Moneda=USD`,
-`TipoCambio=17.3943`, subtotal/IVA/total en USD, `livemode=false`).
+Tests verdes (49 en `test_nota_credito_descuento_relacion_01` + regresión CFDI E 6 / complemento 24 / issue162 9) + ruff/prettier/mkdocs limpios. Experimento sandbox en sitio dev confirmó GL y XML correctos con `Enable Discount Accounting = OFF`.
 
 ---
 
 ## Estado actual
 
-### Ya cerrado (en esta rama)
+### Ya cerrado
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)